ipatests/test_integration/test_ntp_options.py

```diff
@@ -0,0 +1,226 @@
+
+    def test_server_replica_and_client_install_with_ntp_pool_and_ntp_server(
+            self):
+        """
+        test to verify that ipa-server, ipa-replica and ipa-client install with
+        options --ntp-pool and --ntp-server together
+        """
+        ntp_pool = "pool.ntp.org"
+        ntp_server = "1.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+
+        server_install = tasks.install_master(
+            self.master, setup_dns=False,
+            extra_args=['--ntp-pool=%s' % ntp_pool,
+                        '--ntp-server=%s' % ntp_server])
+        assert expected_msg in server_install.stderr_text
+        cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+
+        replica_install = self.install_replica('--ntp-pool=%s' % ntp_pool,
+                                               '--ntp-server=%s' % ntp_server)
+        assert expected_msg in replica_install.stderr_text
+        cmd = self.replica.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+
+        client_install = self.install_client('--ntp-pool=%s' % ntp_pool,
+                                             '--ntp-server=%s' % ntp_server)
+        assert expected_msg in client_install.stderr_text
+        cmd = self.client.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_pool in cmd.stdout_text
+        assert ntp_server in cmd.stdout_text
+        tasks.uninstall_master(self.replica)
+
+        self.cleanup()
+
+    def test_server_and_client_install_with_mixed_options(self):
+        """
+        test to verify that ipa-server and ipa-client install with
+         --ntp-server and -N options would fail
+        """
+        ntp_server = "1.pool.ntp.org"
+        exp_str = ("error: --ntp-server cannot be used"
+                   " together with --no-ntp")
+        exp_pool_str = ("error: --ntp-pool cannot be used"
+                        " together with --no-ntp")
+
+        args1 = ['ipa-server-install', '-N', '--ntp-server=%s' % ntp_server]
+        server_install = self.master.run_command(args1, raiseonerr=False)
+        assert server_install.returncode == 2
+        assert exp_str in server_install.stderr_text
+
+        args2 = ['ipa-client-install', '-N', '--ntp-server=%s' % ntp_server]
+        client_install = self.client.run_command(args2, raiseonerr=False)
+        assert client_install.returncode == 2
+        assert exp_str in client_install.stderr_text
+
+        args3 = ['ipa-client-install', '-N',
+                 '--ntp-pool=%s' % ntp_server.lstrip('1.')]
+        client_install = self.client.run_command(args3, raiseonerr=False)
+        assert client_install.returncode == 2
+        assert exp_pool_str in client_install.stderr_text
+
+    def test_two_step_replica_install_using_ntp_options(self):
```

Comment:
Please rename method to:
```suggestion
    def test_replica_promotion_with_ntp_options(self):
```